app/services/shopify_import/importers/base_importer.rb

```diff
@@ -1,16 +1,7 @@
 module ShopifyImport
   module Importers
     class BaseImporter
-      def self.import!(**params)
```

Comment:
@vfonic I think that we should have params here. Not only for credentials but also for filters like `created_at_min` etc.